docs/source/deployment/corda-health-checker.rst

```diff
@@ -0,0 +1,162 @@
+
+The tool optionally allows operators to upload the resulting report to a support ticket. JIRA credentials can either be provided through the environment variables JIRA_USER and JIRA_PASSWORD, or through the prompt.
+
+Disabling the Corda Health Survey in Production
+-----------------------------------------------
+
+The tool relies on dedicated Artemis queues to relay configuration and runtime information from the Corda Firewall components. This functionality is enabled by default. After verifying a production deployment, operators are advised to disable the health checking functionality (in order to use the standard Artemis setup for Corda Enterprise) by adding the following entry in the Node configuration file:
+
+.. sourcecode:: shell
+
+	enterpriseConfiguration { healthCheck = false }
+
+And the following entry in the Bridge configuration file:
+
+.. sourcecode:: shell
+
+	healthCheck = false
+
+Running the Corda Health Survey
+-------------------------------
+
+The tool runs with the node installation as its current working directory, e.g.:
+
+.. sourcecode:: shell
+
+	$ cd /opt/corda/
+	$ java -jar corda-tools-health-survey.jar
+
+Alternatively, the base directory and Node/Bridge/Float configuration paths can be specified as command-line arguments:
+
+base-directory or -d, specifying the path to the Node installation.
+node-configuration or -c, specifying the path to the Node configuration file.
+
+These options can be specified together, allowing for setups with Node configuration files under separate directories.
+
+The majority of the checks require the Node (and related Firewall components, if present) to have been started before the tool is run. Please refer to the Corda Enterprise documentation on how to do this.
+
+Other available command-line arguments include:
+
+.. sourcecode:: shell
+
+	--local or -l, verify local Node configuration only without checking Bridge/Float, by default verifies all.
+	--jira or -j, prompts user to upload reports a JIRA ticket, by default skips reporting to JIRA.
+	--exclude-logs or -e, exclude node log files from ZIP report, by default logs are included in ZIP report.
+	--text-format or -t, create report as a single txt file without node log files, by default the output of the tool is packaged in a ZIP file.
+	--timeout or -i, override default timeout for sending messages between Node and Firewall components.
+	--config-validate or -v, validates Bridge/Float configuration files.
+	--bridge-configuration or -b, specifying the path to the Bridge configuration file when used alongside config-validate.
+	--float-configuration or -f, specifying the path to the Float configuration file when used alongside config-validate.
+
+Here is sample output from the tool java -jar corda-tools-health-survey-4.1.20190823.jar -t :
+
+.. sourcecode:: shell
+
+	Corda Health Survey Tool 4.1
+	~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Also add a failure scenario with instructions on how to understand what is broken.